packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -340,10 +331,6 @@ class BottomSheetCell extends Component {
 			styles.icon,
 			styles.iconDark
 		);
-		const clearButtonIconStyle = getStylesFromColorScheme(
-			styles.clearButtonIcon,
-			styles.clearButtonIconDark
-		);
```

Comment:
I noticed the clear icon's color is now using the values from `iconStyleBase` instead of having a specific one as we had with `clearButtonIconStyle`, it would be great if we could double-check that those colors are the good ones.

Reply:
I agree with you here that we may wish to have more fine-grained control over the clear button icon color. This code change here was actually an attempt to consolidate color variables a bit, and reduce some of the styling complexity as the colors for `icon` and `iconDark` were already very similar to the new clear button icon colors. With regard to which colors are the "right" ones, there seems to be a bit of variance throughout the app with matching the true native platform UI (if that is even the desired goal, of course). I did note that there is a bit of a difference between the iOS native clear button color, and how we are implementing it. Here are some examples of BottomSheet.Cell usage (top) and Search Control (bottom). In both screenshots, the native iOS clear button (using [`clearButtonMode`](https://developer.apple.com/documentation/uikit/uitextfield/1619622-clearbuttonmode)) is on the **left**, and the Javascript button is on the **right**:

#### **Link Settings**
Link Settings is an example of the BottomSheet.Cell clear button behavior introduced in this PR. (Native iOS clear button on the left, Javascript button on the right using the [existing `icon` and `iconDark` colors](https://github.com/WordPress/gutenberg/blob/2a4257a3b0aaa7c9554cc59d161b90c433b68501/packages/components/src/mobile/bottom-sheet/styles.native.scss#L145-L151).)

![Artboard](https://user-images.githubusercontent.com/643285/146504206-3b472ada-54e1-436d-a917-2769b20e5630.png)

#### **Search Control**
Search Control implements a different clear button solution, using similar colors. (Native iOS clear button is on the left, Javascript on the right.)

![Artboard](https://user-images.githubusercontent.com/643285/146504520-5f98a23f-436a-4e62-ab1e-d83563f821fb.jpg)

These color differences are more noticeable on iOS because the icon has more surface area, but the result is also very similar on Android. While none of the colors in the examples seem "out of place" or unsympathetic to the design, I think there is room for some styling consolidation as part of the next iteration. I'd be interested to know your thoughts on if there is existing styling guidelines for matching the exact native platform colors when re-implementing platform-specific behavior in Javascript. I think it would be an interesting conversation to start, if not.

Reply:
Great investigation @derekblank! I agree that none of the examples look incoherent in terms of styling, so I'd stick with your approach. However, I'd like to ping here @iamthomasbishop, in case he could provide further insights regarding this topic.

@iamthomasbishop I'd appreciate it if you could take a look at this and confirm the right colors for the clear input button, thank you very much for the help 🙇 !